test/static.test.js

```diff
@@ -592,14 +594,25 @@ t.test('prefix default', t => {
   t.doesNotThrow(() => fastify.register(fastifyStatic, pluginOptions))
 })
 
-t.test('root not found error', t => {
-  t.plan(1)
+t.test('root not found warning', t => {
+  t.plan(3)
   const rootPath = path.join(__dirname, 'does-not-exist')
   const pluginOptions = { root: rootPath }
-  const fastify = Fastify({ logger: false })
+  const msgs = []
+  const destination = concat(data => {
+    msgs.push(JSON.parse(data).msg)
```

Comment:
There is no need to accumulate here as the callback would ever be called once.

Reply:
The idea was to report an error if multiple messages were logged, maybe this is being done incorrectly?

Reply:
You would need to use on(‘data’) for that. concact-stream is called once at the end.

Reply:
I just verified JSON.parse(data) will throw an exception if multiple messages were logged so the updated patch still verifies only a single message is logged.